i6_models/primitives/feature_extraction.py

```diff
@@ -33,6 +36,9 @@ class LogMelFeatureExtractionV1Config(ModelConfiguration):
     num_filters: int
     center: bool
     n_fft: Optional[int] = None
+    periodic: Optional[bool] = True
```

Comment:
I think the type is wrong?
```suggestion
    periodic: bool = True
```